core/src/main/kotlin/entity/interaction/MessageInteraction.kt

```diff
@@ -0,0 +1,17 @@
+package dev.kord.core.entity.interaction
+
+import cache.data.MessageInteractionData
+import dev.kord.common.annotation.KordPreview
+import dev.kord.common.entity.InteractionType
+import dev.kord.common.entity.Snowflake
+import dev.kord.core.Kord
+import dev.kord.core.entity.KordEntity
+import dev.kord.core.entity.User
+
+@KordPreview
+class MessageInteraction(val data: MessageInteractionData, override val kord: Kord): KordEntity {
+    override val id: Snowflake get() = data.id
```

Comment:
This is the interactions's ID, right? Is there any kind of behavior/entity we can get from this?

Reply:
there isn't but since it had an ID and requires Kord for the user object that it would be fine to consider it a KordEntity